File: platforms/python/examples/core/renderer/create_storage_texture.py
from fragmentcolor import Renderer, TextureFormat

r = Renderer()
# Empty storage texture -- positional args: (size, format).
tex = r.create_storage_texture([64, 64], TextureFormat.Rgba)

# Pre-seeded with bytes -- same method, optional data kwarg.
pixels = [0] * (64 * 64 * 4)
tex2 = r.create_storage_texture([64, 64], TextureFormat.Rgba, data=pixels)

File: platforms/web/examples/core/renderer/create_storage_texture.js
import { Renderer, TextureFormat } from "fragmentcolor";

const r = new Renderer();

// Empty storage texture.
const tex = await r.createStorageTexture([64, 64], TextureFormat.Rgba);

// Pre-seeded with bytes.
const pixels = Array(64 * 64 * 4).fill(0);
const tex2 = await r.createStorageTexture([64, 64], TextureFormat.Rgba, pixels);